circle.yml

```diff
@@ -1224,6 +1224,7 @@ jobs:
 
   run-frontend-shared-component-tests-chrome:
     <<: *defaults
+    resource_class: medium
```

Comment:
Do these changes use more resources or where they needing bumped separately?

Reply:
I think this is actually a long overdue oversight.  Tests that are running in a full cypress suite should be at medium, otherwise they push the limits of the 'small' resources.